charts/infra-apps/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: infra-apps
 description: Argo CD app-of-apps config for infrastructure components
 type: application
-version: 0.251.1
+version: 0.251.2
```

Comment:
don't we usually minor bump the chart version here?

Reply:
Good point, I'll adjust the prs